meshiphi/dataloaders/vector/abstract_vector.py

```diff
@@ -1120,8 +1120,8 @@ def meshgrid_from_df(data, data_name_list):
         
         def meshgrid_from_xr(data, data_name_list):
             # Extract out each variable and combine as tuple
-            data_arrays = (data[name].values 
-                           for name in data_name_list)
+            data_arrays = list(data[name].values 
+                                for name in data_name_list)
```

Comment:
Not a big deal but couldn't this just be a list comprehension, instead of casting a generator to a list?